plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly

```diff
@@ -46,7 +46,10 @@
     </f:form>
     <br/>
     <f:form method="post" action="export" name="export">
-      <f:submit name="export" value="${%Export Configuration}"/>
+      <span>
+        <p><a href="view">${%View Configuration}</a></p>
```

Comment:
What's there to stop someone without admin permission to view this page?
```suggestion
        <p><a href="view"><button type="button">View</button></a></p>
```
https://jsfiddle.net/az3nmxfu/1/

Reply:
it would work but its invalid html =/
https://stackoverflow.com/a/2906598

Reply:
Will check on the admin bit

Reply:
Could care less... all modern browsers render it correctly :man_shrugging: 
![sad panda](https://media.giphy.com/media/3e18NPUVzoxzO/giphy-downsized.gif)

Reply:
https://www.zdnet.com/article/microsoft-security-chief-ie-is-not-a-browser-so-stop-using-it-as-your-default/ says it all :laughing:

Reply:
We lost the `${%View Configuration}` My bad for giving a bad suggestion

Reply:
I've fixed it up and updated the screenshot

Reply:
admin bit needs fixing

Reply:
Nice styling :muscle:

Reply:
admin check in place, it doesn't stop the page rendering but they get empty:
![image](https://user-images.githubusercontent.com/21194782/56319475-bd742580-6159-11e9-8c42-f6e9ac6aa2b0.png)


the other way to do it would be servlet rendering of some kind I guess. this should be enough though?